NOTES.UNIX

```diff
@@ -89,3 +93,25 @@
 
     $ ./config --prefix=/usr/local/ssl --openssldir=/usr/local/ssl \
         '-Wl,--enable-new-dtags,-rpath,$(LIBRPATH)'
+
+ If you chose to use --run-path option, just pass additional
+ -Wl,--enable-new-dtags.
+
+ It might be worth noting that some/most ELF systems implement support
+ for relative runtime search path, relative to directory containing
```

Comment:
`for runtime search path relative to the directory containing ...`

(too many relatives otherwise)